themes/SuiteP/css/Dawn/variables.scss

```diff
@@ -964,8 +964,8 @@ $navbar-link-color-hover: $color-50;
 $navbar-main-menu-bg: $color-9;
 $navbar-menu-link-color: $color-80;
 $navbar-menu-link-color-hover: $color-50;
-$navbar-menu-seperator: $color-18;
-$navbar-menu-current-seperator: $color-50;
+$navbar-menu-separator: $color-18;
```

Comment:
Similar to the above, we'd need to keep backwards compatibility here. I think a good potential solution would be something like:

```css
$navbar-menu-seperator: $navbar-menu-separator
```

Perhaps with a note above that it's kept for backwards compatibility.